server/src/auth/passport-config.es6

```diff
@@ -80,6 +79,12 @@ passportConfig.logout = (req, res) => {
       )}&state=${loginGov.params.state}&id_token_hint=${req.user.token}`
     );
   }
+  prev = req.headers.referer;
+  if (prev.indexOf('christmas-trees') > -1) {
```

Comment:
1. The only thing changing based on the referer is the url, so we can remove the duplicated code. 2. Is the referer ALWAYS going to be available?

If this PR fixes the major issues, I'm ok with it why creating future issues, but conceptually  i think we want to be able to specify from which authentication strategy we want to log out. With this logic, if I'm an admin who is also logged in via Login.gov and I try to logout of the admin side, this will log me out of the Login.gov side instead...

Reply:
Yeah, after that push I declared prev inside the method, got rid of it in outer scope and set to const. Haven't pushed yet, still working thru conceptually the best way to do this. I'm not sure how an admin would be logged in via admin eauth and login.gov at the same time given your example. 

Ideally we may want to create different user profiles, i.e. forestAdmin and specialUseAdmin, under current status an admin is an admin is an admin.

Reply:
IRL: A user could be a forest service admin AND buy a SU permit which requires Login.gov authentication.

QA Testing: This will happen all the time since folks will want to see both parts.